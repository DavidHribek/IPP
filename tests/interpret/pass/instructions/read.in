123456789
necoCoNeniTrue
nejakyString