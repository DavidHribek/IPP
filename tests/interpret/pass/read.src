.ippcode18 # READ, vypis 123456789 int
DEFVAR GF@int
READ GF@int int
WRITE GF@int
TYPE GF@int GF@int
WRITE GF@int

DEFVAR GF@bool
READ GF@bool bool
WRITE GF@bool
TYPE GF@bool GF@bool
WRITE GF@bool

DEFVAR GF@string
READ GF@string string
WRITE GF@string
TYPE GF@string GF@string
WRITE GF@string