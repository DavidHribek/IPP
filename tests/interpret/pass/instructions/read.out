123456789
int
false
bool
nejakyString
string